galsim/nfw_halo.py

```diff
@@ -42,7 +42,15 @@ def __init__(self, omega_m=0.3, omega_lam=0.7):
         self.omega_lam = omega_lam
         self.omega_c = (1. - omega_m - omega_lam)
         #self.omega_r = 0
```

Comment:
This has nothing to do with this PR,  but if we are ignoring radiation in most of the code, is there any reason to keep the two commented-out lines that involve ```omega_r```?